packages/x-components/src/x-modules/experience-controls/wiring.ts

```diff
@@ -1,8 +1,20 @@
+import { wireCommit } from '../../wiring';
 import { createWiring } from '../../wiring/wiring.utils';
 
+/**
+ * WireCommit for {@link ExperienceControlsXModule}.
+ *
+ * @internal
+ */
+export const controlEvents = wireCommit('setControls');
```

Comment:
This wire is not namespaced so it's not linked to any module (will try to commit a mutation in the root of the store instead to `x/experienceControls/setControls`, which I think is the intention).

What do you want to do with this wire?